core/controller/cluster/controller_operation.go

```diff
@@ -296,6 +305,16 @@ func (c *controller) Rollback(ctx context.Context,
 		return nil, err
 	}
 
+	// 7. update template and tags in db
```

Comment:
Add TODO? 
here is not atomic, and test if should update? and if update error, the rollback should be ok,just print an errlog?

Reply:
fixed